tests/unit/test_github_enterprise.py

```diff
@@ -52,7 +52,7 @@ def url_for_user(self, path=''):
         return self.base_url + 'users/' + example_data['login'] + path
 
     def url_for_admin(self, path=''):
-        return self.base_url + 'admin/users/' + str(example_data['id']) + path
+        return self.base_url + 'admin/users/' + str(example_data['login']) + path
```

Comment:
This line is too long according to our tests: https://travis-ci.org/sigmavirus24/github3.py/jobs/178237362#L177

Please fix this